container/libcontainer/handler.go

```diff
@@ -477,6 +477,9 @@ func scanAdvancedTcpStats(advancedStats *info.TcpAdvancedStat, advancedTcpStatsF
 			}
 		}
 	}
+	if scanner.Err() != nil {
```

Comment:
I believe we do this to return best-effort results if the error occurs mid-way through scanning.